core/src/test/java/org/apache/calcite/test/SqlValidatorTest.java

```diff
@@ -11094,6 +11094,18 @@ private void checkCustomColumnResolving(String table) {
         .fails("(?s).*Cannot apply.*");
   }
 
+  @Test public void testExtractValue() {
```

Comment:
remove this test. you have not changed the validator. the operator test can cover this.

Reply:
Done :)